opensrp-reveal/src/main/java/org/smartregister/reveal/viewholder/AvailableOfflineMapViewHolder.java

```diff
@@ -0,0 +1,26 @@
+package org.smartregister.reveal.viewholder;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.TextView;
+
+import org.smartregister.reveal.R;
+
+public class AvailableOfflineMapViewHolder extends RecyclerView.ViewHolder {
+
+    private Context context;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'context' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4217760)